docs/pages/slimbanner.js

```diff
@@ -0,0 +1,472 @@
+          title="Labels"
+        />
+      </MainSection>
+
+      <MainSection
+        name="Localization"
+        description={`Remember to localize \`message\`, \`helperLink\` and \`iconAccessibilityLabel\`.
+`}
+      />
+      <MainSection name="Variants">
+        <MainSection.Subsection
+          description={`Neutral SlimBanners are intended for Pinner interfaces where Pins and Boards take precedent or where we want to limit the use of color in the design.
+`}
+          title="Neutral"
+        >
+          <MainSection.Card
+            cardSize="lg"
+            defaultCode={`
+<SlimBanner message="Your total audience includes all users who have seen or engaged with any of your Pins in the last 30 days."/>
+`}
+          />
+        </MainSection.Subsection>
+        <MainSection.Subsection
+          description="Info SlimBanners communicate helpful messages or guidance to users about a feature or section."
+          title="Info"
+        >
+          <MainSection.Card
+            cardSize="lg"
+            defaultCode={`
+<Flex direction="column" gap={5} width="100%">
+  <SlimBanner type="info" iconAccessibilityLabel="Info" message="Idea Pins are now available across platforms."/>
+  <SlimBanner type="infoBare" iconAccessibilityLabel="Info" message="Idea Pins are now available across platforms."/>
+</Flex>
+`}
+          />
+        </MainSection.Subsection>
+        <MainSection.Subsection
+          description="Success SlimBanners communicate confirmation regarding an action within a larger flow."
+          title="Success"
+        >
+          <MainSection.Card
+            cardSize="lg"
+            defaultCode={`
+<Flex direction="column" gap={5} width="100%">
+  <SlimBanner type="success" iconAccessibilityLabel="Info" message="Your ads are doing great! Keep it up by using recommendations to optimize your ad spend."/>
+  <SlimBanner type="successBare" iconAccessibilityLabel="Info" message="Your ads are doing great! Keep it up by using recommendations to optimize your ad spend."/>
+</Flex>
+`}
+          />
+        </MainSection.Subsection>
+        <MainSection.Subsection
+          description="Warning SlimBanners communicate cautionary messages to users. The SlimBanner should provide clear guidance on how to correct an issue and/or learn more about it. This is done via a link inside of the banner, or clear actions in the section that the banner refers to."
+          title="Warning"
+        >
+          <MainSection.Card
+            cardSize="lg"
+            defaultCode={`
+<Flex direction="column" gap={5} width="100%">
+  <SlimBanner
+    type="warning"
+    iconAccessibilityLabel="Info"
+    message="This feature is being sunset and will not be available after May 1, 2024."
+    helperLink={{
+      text: "Learn more.",
+      accessibilityLabel: "Learn more Pinterest.com",
```

Comment:
Nit: "learn more about deprecated features" maybe? We shouldn't encourage putting the URL in the aria label

Reply:
DONE